GoogleForCreators/web-stories-wp
# Redesign Canvas (Moveable) selection

## Context



## Summary



## Relevant Technical Choices



## To-do



## User-facing changes
Canvas selection has now been updated to [Figma](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=1009%3A4323).


## Testing Instructions
See the issue [description](https://github.com/google/web-stories-wp/issues/6200) with the todo list and verify style matching based on Figma.

Before:
![Screenshot 2021-02-08 at 11 33 09](https://user-images.githubusercontent.com/3294597/107201287-1250f200-69f9-11eb-97ee-482680f82331.png)

After:
![Screenshot 2021-02-08 at 11 32 35](https://user-images.githubusercontent.com/3294597/107201304-1715a600-69f9-11eb-8f11-48ded3307057.png)




### QA






This PR can be tested by following these steps:

1. Add an element, e.g. a square shape.
2. Select the element
3. Verify the selection matching the style in Figma.

### UAT






This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6200
